### ISSUE TITLE
Improve text rendering quality when using the BitmapWriter

### ISSUE THREAD

**Issue by eseifert** [eseifert/gral#110]
The default text quality when producing an PNG (or JPEG) file using the `BitmapWriter` is not great, sometimes barely readable.

This can be improved by passing `RenderingHints`.

The current code in `gral-core/src/main/java/de/erichseifert/gral/io/plots/BitmapWriter.java` uses this:

```
DrawingContext context = new DrawingContext((Graphics2D)image.getGraphics());
```

This can be improved using this instead:

```
Graphics2D graphics = (Graphics2D) image.getGraphics();
graphics.setRenderingHint(RenderingHints.KEY_ANTIALIASING,
RenderingHints.VALUE_ANTIALIAS_ON);
graphics.setRenderingHint(RenderingHints.KEY_RENDERING,
RenderingHints.VALUE_RENDER_QUALITY);
graphics.setRenderingHint(RenderingHints.KEY_INTERPOLATION,
RenderingHints.VALUE_INTERPOLATION_BICUBIC);
graphics.setRenderingHint(RenderingHints.KEY_FRACTIONALMETRICS,
RenderingHints.VALUE_FRACTIONALMETRICS_ON);
DrawingContext context = new DrawingContext(graphics);
```

I'm attaching a couple of files to illustrate the difference (`test1.png` is produced with version 0.10 and `test2.png` is produced with this change).

 _Imported from trac ticket #111,  created by bruno on 05-18-2014 at 22:02, last modified: 05-20-2014 at 14:21_

**Comment by eseifert** [comment]
Thanks for the report. I'll incorporate the proposed settings as soon as I can.

_Trac comment by eseifert on 05-20-2014 at 14:21_

**Comment by seifertm** [comment]
Issue addressed in commit 14a46bf8a5244a4f3537e6f31924e5ed3ff6d648.
